applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_element.hpp

```diff
@@ -276,8 +276,11 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwSmallStrainElement : public UPwBa
     double CalculateBiotCoefficient(const ElementVariables& rVariables, bool hasBiotCoefficient) const;
 
     virtual Vector CalculateGreenLagrangeStrain(const Matrix& rDeformationGradient);
-    virtual void   CalculateCauchyStrain(ElementVariables& rVariables);
-    virtual void   CalculateStrain(ElementVariables& rVariables, unsigned int GPoint);
+    virtual Vector CalculateCauchyStrain(const Matrix& rB, const Vector& rDisplacements);
+    virtual Vector CalculateStrain(const Matrix& rDeformationGradient,
+                                   const Matrix& rB,
+                                   const Vector& rDisplacements,
+                                   bool          UseHenckyStrain);
```

Comment:
I would suggest to make these member functions `const`, to express that they won't modify the element's state.

Reply:
Done